src/components/Player/PlayerList.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import PlayerList from './PlayerList';
+import { BrowserRouter } from 'react-router-dom';
+
+test('renders playerdetail', () => {
```

Comment:
Making this test async and awaiting with findByText would allow the Component's useEffect to run and render all of the data.